src/styles/app.css

```diff
@@ -1,134 +1,312 @@
+  background-color: #f5f6fa;
+}
+
+.panel-bg {
+  width: 87.5rem;
+  height: 68.563rem;
+  margin: 0 2.5rem 4.063rem 0;
+  padding: 0 0 49.625rem;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #fafafa;
+}
+
+.panel {
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+
+.panel h2 {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: Montserrat;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.left {
+  display: flex;
+  width: 70%;
+}
+
+.left .first {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: Montserrat;
+  font-size: 0.813rem;
+  font-weight: normal;
+}
+
+.left .seconde {
+  width: 6.563rem;
+  height: 1rem;
+  margin: 1.125rem 21.5rem 0.688rem 2.563rem;
+  opacity: 0.5;
+  font-family: Montserrat;
+  font-size: 0.813rem;
+}
+
+.oval {
+  width: 4.813rem;
+  height: 4.813rem;
+  margin: 0 0 0 21.5rem;
+  padding: 0.875rem;
+}
+
+.lesson {
   width: 75rem;
   height: 10.625rem;
   margin: 2.313rem 6.25rem 0;
   padding: 2rem 9.188rem 1.625rem 1.688rem;
   border-radius: 4px;
-  border: solid 1px #e8e8e8;
-  background-color: #fff;
+  border: solid 1px #fff;
+  background-color: #e8e8e8;
+  display: flex;
 }
 
-.School-of {
+.action {
   width: 3.063rem;
   height: 1.125rem;
   margin: 0 2.438rem 0 0;
   opacity: 0.5;
+  font-family: Montserrat;
   font-size: 0.875rem;
   font-weight: bold;
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
   letter-spacing: normal;
   color: #121212;
+  display: block;
 }
 
-.Title {
+.title {
   width: 12.313rem;
   height: 1.813rem;
   margin: 0.188rem 20.188rem 0 0;
+  font-family: RobotoSlab;
   font-size: 1.375rem;
   font-weight: bold;
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
   letter-spacing: -0.2px;
   color: #121212;
+  display: block;
 }
 
-.Suzanne-Collins {
+.author {
   width: 6.688rem;
   height: 1.188rem;
   margin: 0.25rem 0 0;
+  font-family: RobotoSlab;
   font-size: 0.875rem;
   font-weight: 300;
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
   letter-spacing: normal;
   color: #4386bf;
+  display: block;
 }
 
-button {
+.but {
+  width: 4.563rem;
+  height: 1.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: RobotoSlab;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
   line-height: normal;
   letter-spacing: normal;
   color: #4386bf;
-  background-color: transparent;
-  border: none;
-  cursor: pointer;
 }
 
-.Comments {
-  width: 4.563rem;
-  height: 1.188rem;
-  margin: 0.188rem 0.938rem 0.125rem 0;
+.line {
```

Comment:
- [ ] Kindly add the highlighted elements in the image below to your design so as to ensure that your app matches the given design

<img width="1233" alt="Screenshot 2023-06-15 at 22 24 41" src="https://github.com/francksefu/booksmr/assets/64774936/8d6c0f3d-205c-4176-8fc4-68bd0fb621e9">